src/ralph/accounts/models.py

```diff
@@ -188,7 +188,7 @@ def permissions_hash(self):
         Hash for caching is calculated from user ID and its permissions.
         """
         perms_set = frozenset(self.get_all_permissions())
-        key = ':'.join((self.id, hash(perms_set)))
+        key = ':'.join((str(self.id), str(hash(perms_set))))
```

Comment:
Hmm I've just tested it and it seems to work on two different users. Maybe change of permissions will not work here (because empty string is injected as hash of permissions every time)?